dbtest/dbserver.go

```diff
@@ -29,6 +29,8 @@ type DBServer struct {
 	server  *exec.Cmd
 	dbpath  string
 	host    string
+	engine  string
+	args    []string
```

Comment:
There's no real reason to have the args as part of the DBServer struct (from what I can see it's only done so you can check them during the tests).
Instead it would be better to check that the right storage engine has been set by using [db.serverStatus().storageEngine](https://docs.mongodb.com/manual/reference/command/serverStatus/#storageengine), which will check the actual storage engine being used instead of the args set previously.

Reply:
Yes, argv only there for testing.

Great idea @eminano, I'll submit a change tomorrow that does this instead.